test/potluck_test.rb

```diff
@@ -0,0 +1,97 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/potluck"
+require "./lib/dish"
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new
+  end
+
+  def test_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_dishes
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_dishes_can_add
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck.add_dish(couscous_salad)
+
+    refute potluck.dishes.empty?
```

Comment:
This works, but can also actually make an assertion about the contents of the `dishes` array.

```
assert_equal [couscous_salad], potluck.dishes
```